src/main/java/org/opensearch/neuralsearch/processor/TextChunkingProcessor.java

```diff
@@ -170,9 +172,11 @@ public IngestDocument execute(final IngestDocument ingestDocument) {
         // fixed token length algorithm needs runtime parameter max_token_count for tokenization
         Map<String, Object> runtimeParameters = new HashMap<>();
         int maxTokenCount = getMaxTokenCount(sourceAndMetadataMap);
+        int stringTobeChunkedCount = getStringTobeChunkedCountFromMap(sourceAndMetadataMap, fieldMap);
```

Comment:
Can we rename this to `chunkFieldsCount`?

Reply:
A single chunking field may contain multiple strings like:
```
{
  "body": ["string 1", "string 2", "string 3"]
}
```

Reply:
Should be named as `chunkStringCount`